### ISSUE TITLE
Labels in LINT.IfChange() aren't detected in cross references

### ISSUE THREAD

**Issue by michael-delphos** [ebrevdo/ifttt-lint#8]
Hi!

Thanks for making this available.

I'm having trouble, in a Python script, getting cross referenced labels to recognize each other.

Here's my example

```
# file1.py

# LINT.IfChange("label")
pass
# LINT.ThenChange("relative/path/file2.py#other")
```

```
# file2.py

# LINT.IfChange("other")
pass
# LINT.ThenChange("relative/path/file1.py#label")
```

Then I get this when running the tool

```shell
git diff HEAD~1 | npx ifttt-lint  
```
```
[ifttt] file1.py-> ThenChange 'file2.py' (line X): label 'other' not found in 'file2.py'. Available labels: none
```

I have a work around, but it requires making the label explicit

```
# LINT.Label("label")
# LINT.IfChange
```

I appreciate all your work on this!
